PR for SRVLOGIC-524: Add the section to upgrade Serverless Logic Operator from 1.34.0 to 1.35.0 (#90790)
openshift/openshift-docs: modules/serverless-logic-upgrading-osl-operator.adoc

:_mod-docs-content-type: PROCEDURE
[id="serverless-logic-upgrading-osl-operator_{context}"]
= Upgrading the {ServerlessLogicOperatorName}

To transition from OSL version 1.34.0 to 1.35.0, you must upgrade the {ServerlessLogicOperatorName} using the OpenShift web console. This upgrade ensures compatibility with newer features and proper functioning of your workflows.

.Prerequisites

* You have access to the cluster with `cluster-admin` privileges.
* You have {ServerlessLogicOperatorName} installed on your cluster.
* You have access to a {ServerlessLogicProductName} project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
* You have access to the *OpenShift Management Console* for Operator upgrades.
* You have installed the OpenShift CLI (`oc`).

.Procedure

. In the web console, navigate to the *Operators* -> *OperatorHub* -> *Installed Operator* page.
. Select the `openshift-serverless-logic` namespace from the *Installed Namespace*.
. In the list of installed operators, find and click the {ServerlessLogicOperatorName}.
. In the Operator details page, click on the *Subscription* tab, Click *Edit Subscription*.

> . In the *Update* channel, select, `stable-1.35` and click *Save* to apply the changes.

Reviewer: ```suggestion
. In the *Upgrade status* options, click on the `Upgrade available`  link
. Finally click on the *Preview install plan* and *Approve* it to start the update
```

Reviewer: In OSL we don't have this stable-1.35, stable-1.xx, channels.
So we change a bit these  steps.

. To monitor the upgrade process, execute the following command: 
+
[source,terminal]
----

> $ oc get subscription serverless-logic-operator -n openshift-serverless-logic -o jsonpath='{.status.currentCSV}'

Reviewer: ```suggestion
oc get subscription logic-operator-rhel8 -n openshift-serverless-logic -o jsonpath='{.status.installedCSV}'
```

Author: Updated! I am keeping the *$* sign as per the style guide. If we mention any command, we must put the *$* before it.

----
+
.Expected output
[source,terminal]
----

> $ serverless-logic-operator.v1.35.0

Reviewer: ```suggestion
$ logic-operator-rhel8.v1.35.0
```
